hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -177,6 +177,9 @@ public Operation convert(String value) throws ParameterException {
     @Parameter(names = {"--table-type"}, description = "Type of table. COPY_ON_WRITE (or) MERGE_ON_READ", required = true)
     public String tableType;
 
+    @Parameter(names = {"--base-file-format"}, description = "File format for the base files. PARQUET (or) HFILE", required = false)
```

Comment:
Similar functionality needs to be done for Spark SQL Writer. See HoodieSparkSqlWriter.scala

Reply:
Done. The changes are in 
hudi-spark/src/main/java/org/apache/hudi/DataSourceUtils.java
hudi-spark/src/main/scala/org/apache/hudi/DataSourceOptions.scala
hudi-spark/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala